Client/Frontend/Settings/BraveShieldsAndPrivacySettingsController.swift

```diff
@@ -264,6 +273,16 @@ class BraveShieldsAndPrivacySettingsController: TableViewController {
   }()
 
   private lazy var otherSettingsSection: Section = {
+    let p3aRow = Row.boolRow(
+      title: Strings.P3A.settingTitle,
+      detailText: Strings.P3A.settingSubtitle,
+      toggleValue: p3aUtilities.isP3AEnabled,
+      valueChange: { [unowned self] isOn in
```

Comment:
do we need `self` here? it's not used

Reply:
Yes `p3aUtilities`is global and if you will remove this it will give a compilation error